samples/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#dependencies
+
+find_package(Threads REQUIRED)
```

Comment:
Threads is required here, why ?

Reply:
Not required forgot to remove this from the CMakeLists was just testing something. Removing it.